### PR TITLE
basic doc for identity API

### PR DESCRIPTION
**Pull request by lperrin**
Basic doc for chat identity API.

### CODE REVIEW

File: source/chat.html.md [hunk 1]
```diff
@@ -64,3 +64,37 @@ It will return an `unbind` function that you can call to unregister the handler.
 ```javascript
   const unbind = FrontChat('onUnreadChange', handler);
 ```
+
+## FrontChat('identity', options)
```
Comment (seanxiesx) [inline source/chat.html.md]: i gather we won't document the ability to do this through FrontChat('init') as well to start?

Comment (lperrin) [reply source/chat.html.md]: Good point, fixed

**Comment by lperrin** [comment]
@xdmnl this is the PR with the doc for Chat identity. If it looks OK to you, can you merge and deploy? (or tell me how to do it, but I didn't manage to do it last time).

File: source/chat.html.md (hunk 1 above)
Comment (lperrin) [reply source/chat.html.md]: fixed

**Pull request merged**